onert-micro/luci-interpreter/pal/mcu/PALAdd.h

```diff
@@ -28,8 +28,9 @@ namespace luci_interpreter_pal
 // TODO: check if there real activation value
 template <typename T>
 inline void Add(const ArithmeticParams &params, const int flat_size, const T *input1_data,
-                const T *input2_data, T *output_data)
+                const T *input2_data, T *output_data, luci_interpreter::OperationGraphStatus status)
 {
+  assert(status == luci_interpreter::OperationGraphStatus::USUAL);
```

Comment:
add assert in kernels supported this new feature to check that in `pal/mcu` all use USUAL